### DISCUSSION TITLE
20240808 Weekly Development Meeting

### DISCUSSION THREAD

**Discussion by josecelano** [nautilus-cyberneering/.github#105] [Meetings]
- Time:  9:30 in Canary Islands (WET (UTC+0)).
- Time: 10:30 in Vaduz (CET (UTC+1)).

Location:
<https://meeting.blockfinance-eco.li/NautilusCyberneeringWeeklyDevMeeting>

Expected duration: 2 to 4 hours with breaks

Communicated attendants:

- Cameron Garnham(@da2ce7)  - x 
- Jose Celano(@josecelano) - x
- Constantin Bosse(@cgbosse) - x
- Mario Melchor(@mario-nt) - x
- Graeme Byrne(@grmbyrn) - x

-------------------------------------------------------------

## Agenda:

1. Everyone will take their turn and:
- Go over his notes added as comments for the meeting which will have been added in advance to give an update of all the work done during the previous week indicating.
	- The issues worked on
	- The progress made on them
	- Other things done
- Inform of any faced challenges and request help if necessary to work on specific issues.
- Highlight the encountered difficulties, especially if there is a risk of them causing delays or affecting end product's quality.
- Communicate what will be the next week's tasks.
- List any new topics for further discussion in an additional Technical meeting that he noted for future discussion in comments to this discussion either before the meeting or after his turn so that they get considered.

During the meeting every team member will take their turns and follow this sequence.
The topics for further discussion will be noted in advance by each the team member and to the meetings agenda.

2. At the end of the meeting if there any new technical topics to be discussed further it will be decided to
set another meeting date for the discussion of these or to hold the meeting hereafter.

Also the persons to be part of the meeting will be decided on so that the others will be free to continue their own work.

**Comment by josecelano** [comment]
### Torrust Tracker

Issues:

Overhaul Tracker Keys:
  - https://github.com/torrust/torrust-tracker/issues/989
  - https://github.com/torrust/torrust-tracker/issues/978

Config Overhaul:
- https://github.com/torrust/torrust-tracker/issues/958

Issues **(needs feedback)**:
- https://github.com/torrust/torrust-tracker/issues/811

Comment on issues:

- https://github.com/torrust/torrust-tracker/issues/997

PRs:

- https://github.com/torrust/torrust-tracker/pull/954

### Torrust Index

Issues:

- https://github.com/torrust/torrust-index/issues/702

### Torrust Index GUI

Issues:

Update config files:
- https://github.com/torrust/torrust-index-gui/issues/591
- https://github.com/torrust/torrust-index-gui/issues/589

Bugs or small improvements:
- https://github.com/torrust/torrust-index-gui/issues/592
- https://github.com/torrust/torrust-index-gui/issues/593
- https://github.com/torrust/torrust-index-gui/issues/600
- https://github.com/torrust/torrust-index-gui/issues/601

Feature request:
- https://github.com/torrust/torrust-index-gui/issues/599 **(needs feedback)**

### Torrust Demo

Issues:

- https://github.com/torrust/torrust-demo/issues/10

### Torrust Compose

Issues:

- https://github.com/torrust/torrust-demo/issues/28

### Aquatic Bencher

PRs:

- https://github.com/greatest-ape/aquatic/pull/208

**Comment by grmbyrn** [comment]
### Torrust Website

[make changes as recommended by review #227](https://github.com/torrust/torrust-website/pull/227)
* Insert text missing from the Miro Board
* Populate recent posts section with the actual links and actual text excerpts for blog posts
* Add links to Rust and Vue in Torrust Index and Torrust Tracker sections
* Insert the links in the individual boxes under Key Features
* Insert the links in the individual boxes under Stack
* Add data to table and make it scrollable
* Insert the links in the individual boxes under Key Features
* Correct the placement of the Performance section

**Comment by mario-nt** [comment]
### Index
	

- https://github.com/torrust/torrust-index/pull/662
- https://github.com/torrust/torrust-index/issues/615 - Attribute based authorization
- https://github.com/torrust/torrust-index/issues/615 - tests

### Other

- Review pending emails and github notifications
